app/components/highlight/highlight.js

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import * as highlightActionCreators from 'actions/highlight'
+//import './styles.scss';
+
+const assembledActionCreators = Object.assign({}, highlightActionCreators);
+
+const mapStateToProps = state => {
+  return {
+    highlights: state.highlight.highlights,
+    selectedHighlight: state.highlight.selectedHighlight
+    };
+}
+
+const HighlightModule = React.createClass({
+  displayName: 'HighlightModule',
+
+  propTypes: {
+    text: React.PropTypes.string.isRequired,
+    topics: React.PropTypes.array.isRequired,
+    highlights: React.PropTypes.array.isRequired,
+    currentTopic: React.PropTypes.string.isRequired,
+    selectedHighlight: React.PropTypes.array.isRequired,
+    colors: React.PropTypes.array.isRequired,
+  },
+
+  /*
+  Domain: current stored highlight objects
```

Comment:
comments should be per method. best practice specifies stating params and return vals and a short description. big summaries tend to not be read. please see this styleguide for suggestions: https://github.com/airbnb/javascript#comments